orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/Front50Service.groovy

```diff
@@ -68,6 +68,12 @@ interface Front50Service {
   @GET("/pipelines?restricted=false")
   List<Map<String, Object>> getAllPipelines()
 
+  @POST('/actions/pipelines/reorder')
+  Response reorderPipelines(@Body Map<String, Object> body)
```

Comment:
```suggestion
  Response reorderPipelines(@Body Map<String, Integer> body)
```

Reply:
Since the request body has fields for both `idsToIndices` and `application` I ended up taking Jake's advice and creating a small helper class for this.

Reply:
👍